### PR TITLE
LPS-192406 UI experience improvement in the Content Dashboard info panel

### PR DESCRIPTION
**Pull request by Stephy6**
# **Motivation**
The motivation behind this ticket is to improve Content Dashboard info panels UI experience.
https://liferay.atlassian.net/browse/LPS-164349
https://liferay.atlassian.net/browse/LPS-164348

# **Proposed Solution**
What I did to solve this problem is to change the Details key name to Properties in one of the info panels section and make that section expanded by default when the info panel is open. Also I moved the Categorization section to a new tab by using a new ClayTab.Item and ClayTab.Content when the item has linked vocabularies or tags.

# **Steps to Verify**
2. Create some tags and vocabularies.
3. Create a Display Page Template for web contents.
4. Create a web content and add the tag and vocabularies to it and also attach it to the DPT created.
5. Go to the Content Dashboard.
6. Hover over the web content created and click on the info button in the right side
7. When the info panel is opened, check that the item contains three tabs and all work correctly
8. Also, check the same with some item that hasn't has categories

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView/ItemVocabularies.js [hunk 1]
```diff
@@ -25,13 +26,7 @@ const ItemVocabularies = ({cssClassNames = '', title, vocabularies}) => {
...
 
 	return (
-		<div
-			className={`c-mb-4 item-vocabularies sidebar-section ${cssClassNames}`}
-		>
-			<h6 className="font-weight-semi-bold sidebar-section-subtitle-sm text-2 text-secondary text-uppercase">
-				{title}
-			</h6>
-
+		<CollapsibleSection expanded title={Liferay.Language.get(title)}>
```
Comment (victorg1991) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView/ItemVocabularies.js:29]: You cannot do this :(

Liferay.Language.get is a fake function, you cannot use variables in it, you should use directly the string you want to translate. This is why there is a filter in the portal that changes all the Liferay.Language.get references and translates them so this code doesn't even get to the front

we should pass the title translated to this component instead

File: modules/apps/content-dashboard/content-dashboard-web/test/js/components/SidebarPanelInfoView/SidebarPanelInfoView.test.js [hunk 2]
```diff
@@ -183,22 +199,38 @@ describe('SidebarPanelInfoView', () => {
...
+		);
 
-		const subtitles = container.querySelectorAll(
-			'.item-vocabularies h6.sidebar-section-subtitle-sm'
+		fireEvent(
+			getByText('categorization'),
+			new MouseEvent('click', {
+				bubbles: true,
+				cancelable: true,
+			})
```
Comment (victorg1991) [inline modules/apps/content-dashboard/content-dashboard-web/test/js/components/SidebarPanelInfoView/SidebarPanelInfoView.test.js:211]: There is a library called `userEvent` which is done specifically for this :) 

you can import it like:

```javascript
import userEvent from '@testing-library/user-event';
```

and simply do:

```javascript
userEvent.click(getByText('categorization'));
```

Comment (victorg1991) [reply modules/apps/content-dashboard/content-dashboard-web/test/js/components/SidebarPanelInfoView/SidebarPanelInfoView.test.js:211]: Could you update that ? :)

**Comment by victorg1991** [comment]
Just a small detail @Stephy6  :D

**Comment by Stephy6** [comment]
ci:forward

**Comment by victorg1991** [comment]
Great job :D

**Comment by victorg1991** [comment]
ci:forward

**Comment by Stephy6** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:test:relevant

**Comment by Stephy6** [comment]
ci:forward

**Comment by victorg1991** [comment]
image

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**